lms/djangoapps/branding/tests/test_page.py

```diff
@@ -305,3 +306,54 @@ def test_get_programs_with_type_called(self):
                 response = self.client.get(url)
                 self.assertEqual(response.status_code, 200)
                 mock_get_programs_with_type.assert_called_once()
+
+
+@attr(shard=1)
+@patch.dict(settings.FEATURES, {"ENABLE_JOURNAL_INTEGRATION": True})
+class MarketingPageJournalsTests(SiteMixin, ModuleStoreTestCase):
```

Comment:
Question: Does this test need to live in this module?  Or can it be co-located with the `journals` app?

Reply:
moved to journals app